grails-app/services/au/org/ala/profile/hub/ImageService.groovy

```diff
@@ -378,7 +378,7 @@ class ImageService {
         response.resp.count = numberOfPublishedImages + numberOfLocalImages
 
         if (!readonlyView) {
-            response.resp.availImagesCount = publishedImagesMap?.resp?.totalRecords + numberOfLocalImages
+            response.resp.availImagesCount = (publishedImagesMap.resp?publishedImagesMap?.resp?.totalRecords:0) + numberOfLocalImages
```

Comment:
It is most likely that `publishedImagesMap.resp` will have data. But it might have properties like `error` and not have `totalRecords`. This will not fix such cases.
```
publishedImagesMap.resp = [ "error": "server error", "status": 500]
```

Reply:
There is no resp attribute in error case
That's why publishedImagesMap.resp? is getting null when error occurred